Docs/v1.11 (#220)
TrossenRobotics/trossen_arm: docs/getting_started/configuration.rst

Ranges: :math:`\mathbb{R}`

Links
-----

New in version :ref:`changelog:1.11.0`.

The links define the inertial properties of every link of the arm: the base link, every arm link, the palm, and both finger links, in that order.

> The Arm Controller feeds them to its inverse dynamics model, so they determine how well gravity and inertia are compensated.

Reviewer: ```suggestion
The Arm Controller feeds the properties to its inverse dynamics model, so they specify how gravity and inertia are compensated.
```

The definition of :class:`trossen_arm::Link` follows the `URDF convention <https://wiki.ros.org/urdf/XML/link>`_, same as the :ref:`getting_started/configuration:link properties` of the end effector.

.. tip::

    Commonly used links are provided in :class:`trossen_arm::StandardLinks`.
    Check out the :ref:`getting_started/demo_scripts:`set_links`_` demo for details.

.. note::

    The last three links, i.e., the palm and both finger links, are overwritten from the :ref:`getting_started/configuration:end effector` configuration whenever the driver configures.
    Set those three through the end effector instead, and use this configuration for the base link and the arm links.

.. note::

>     The standard links are nominal values from the CAD model of an arm variant, so they only approximate the arm at hand.

Reviewer: Recommend language similar to this

```suggestion
    The standard links are nominal values from the CAD model of an arm variant, so they only approximate any specific arm.
```
